### PR TITLE
[MMCA-5016] - Update the API specs as per updated ACC44 & ACC45

### PR DESCRIPTION
**Pull request by tyagikumarvineet**
Description - Update the schemas for ACC44 and ACC45

Below have been covered as part of this PR

- [x] update the schemas
- [x] rename the schemas' file after latest schema version
- [x] update a test case for invalid request

https://jira.tools.tax.service.gov.uk/browse/MMCA-5016

### CODE REVIEW

File: app/utils/JSONSchemaValidator.scala [hunk 1]
```diff
@@ -36,9 +36,9 @@ class JSONSchemaValidator {
   val ssfnRequestSchema = "/schemas/statement-search-failure-notification-request-schema.json"
   val ssfnErrorResponseSchema = "/schemas/statement-search-failure-notification-error-response-schema.json"
   val ssfnSecureMessageRequestSchema = "/schemas/secure-message-request-schema.json"
-  val acc44RequestSchema = "/schemas/acc44-fe-request-schema-v1.0.json"
-  val acc44ResponseSchema = "/schemas/acc44-fe-response-schema-success-v1.0.json"
-  val acc45RequestSchema = "/schemas/acc45-fe-request-schema-v1.0.json"
+  val acc44RequestSchema = "/schemas/acc44-fe-request-schema-v1.2.json"
+  val acc44ResponseSchema = "/schemas/acc44-fe-response-schema-success-v1.2.json"
+  val acc45RequestSchema = "/schemas/acc45-fe-request-schema-v1.1.json"
```
Comment (magicaljellybeans) [inline app/utils/JSONSchemaValidator.scala:41]: Up to you - these values could go in a companion object to be imported rather than injected.

**Review by harshare — APPROVED** [review]
LGTM

**Review by AdamCoops — APPROVED** [review]
seems good!

**Review by magicaljellybeans — APPROVED** [review]
Just one minor comment that is up to you.

File: app/utils/JSONSchemaValidator.scala (hunk 1 above)
Comment (tyagikumarvineet) [reply app/utils/JSONSchemaValidator.scala:41]: as discussed over the call, we can take this as part of separate tech debt task as this file is being used at quite a few places

**Pull request merged**